pyproject.toml

```diff
@@ -203,7 +202,7 @@ max-complexity = 54  # Recommended: 10
 [tool.ruff.lint.per-file-ignores]
 "_cryptography.py" = ["S304", "S305"]  # Use of insecure cipher / modes, aka RC4 and AES-ECB
 "_encryption.py" = ["S324"]
-"_writer.py" = ["S324"]
+"_writer.py" = ["ARG005", "S324"]
```

Comment:
As already mentioned in your other PR, please refrain from adding file-wide exceptions like this if it can be avoided. Why not use `noqa` for the actual cases where we want to ignore it?